composer.json

```diff
@@ -34,8 +31,7 @@
         "doctrine/phpcr-odm": "To enable support for the PHPCR ODM documents (^1.4)",
         "doctrine/phpcr-bundle": "To enable support for the PHPCR ODM documents",
         "doctrine/orm": "To enable support for the ORM entities (^2.5)",
-        "symfony-cmf/content-bundle": "To optionally use the configured value for 'content_basepath' from the CoreBundle",
-        "symfony-cmf/core-bundle": "To use the RouteTypeType form type and for easier configuration (^2.0)"
```

Comment:
I am aware that this form type exists, but I can't find how it's related to the CoreBundle: https://github.com/symfony-cmf/routing-bundle/blob/master/src/Form/Type/RouteTypeType.php